### PR TITLE
Refactor input information and add docs about mp4s

### LINKED ISSUE #391: Document MP4 inputs
Add missing documentation for mp4 inputs

### PR DESCRIPTION
**Pull request by jerzywilczek**
Closes #391

### CODE REVIEW

File: docs/pages/api/inputs/rtp.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Docs from "@site/pages/api/generated/renderer-RtpInputStream.md"
+
+# RTP
+RTP streams allow streaming video and audio to the compositor.
```
Comment (wkozyra95) [inline docs/pages/api/inputs/rtp.md:4]:
```suggestion
An input type that allows streaming video and audio to the compositor over RTP.
```

File: docs/pages/api/inputs/mp4.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Docs from "@site/pages/api/generated/renderer-Mp4.md"
+
+# MP4
+MP4 inputs allow the compositor to use video and audio from MP4 files in scenes.
```
Comment (wkozyra95) [inline docs/pages/api/inputs/mp4.md:4]:
```suggestion
An Input type that allows the compositor to read static MP4 files.
```

Comment (wkozyra95) [reply docs/pages/api/inputs/mp4.md:4]: Also add information what is supported, what choices we are making when there is an option. e.g.
- what codecs we are supporting (audio + video)
- how we are selecting tracks when there is more than one
...

File: docs/sidebars.ts [hunk 3]
```diff
@@ -66,6 +66,13 @@ const sidebars: SidebarsConfig = {
           description: 'Resources that need to be registered first before they can be used.',
           items: ['api/renderers/shader', 'api/renderers/image', 'api/renderers/web'],
         },
+        {
+          type: 'category',
+          label: 'Inputs',
+          collapsible: false,
+          description: 'Inputs to the rendering pipeline',
```
Comment (wkozyra95) [inline docs/sidebars.ts:73]:
```suggestion
          description: 'Elements that deliver media from external sources.',
```

File: docs/pages/api/routes.md [hunk 4]
```diff
@@ -43,64 +43,14 @@ type UpdateScene = {
...
-  audio?: Audio;
+  entity_type: "rtp_input_stream" | "mp4";
+  ... // input specific options
 }
 ```
+See inputs documentation to learn more.
 
-Parameters of registered RTP input stream. Before using input in video composition or output mixing, input has to be firstly registered using `register_input` request.
-
-At least one of `video` and `audio` has to be defined.
```
Comment (wkozyra95) [inline docs/pages/api/routes.md:56]: make sure that the generated descriptions are on par with what was here. It does not have to be the same, but check if everything makes sense. It's possible that some fixed landed here in the past, but code comments were not updated

**Review by wkozyra95 — APPROVED** [review]

**Review by noituri — APPROVED** [review]

**Review by WojciechBarczynski — APPROVED** [review]

**Pull request merged**